input/fsh/Terminology.fsh

```diff
@@ -1,34 +1,36 @@
-// DEMOGRAPHICS CodeSystems
-
 Alias: $FHIRMS = http://terminology.hl7.org/CodeSystem/v3-MaritalStatus
+Alias: $SCT = http://snomed.info/sct // standard set is published separately for re-use in other IGs
+
+///////////////////////////////////////////////////// DEMOGRAPHICS CodeSystems \\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\\
```

Comment:
Interesting idea, but it makes it harder to work with because the search results are shifted:

![image](https://user-images.githubusercontent.com/110988/170922648-79b6cd75-1a11-4a43-916b-44edf565a917.png)

Let's just stick to this:

// Demographics CodeSystems